### PR TITLE
Walkthroughs / guides

### PR DESCRIPTION
**Pull request by Noviny**

### CODE REVIEW

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+```sh
+mkdir keystone-learning
+cd keystone-learning
+yarn init
+```
+
+After this, let's add the keystone package
+
+`yarn add @keystone-6/keystone`
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:21]:
```suggestion
```
yarn add @keystone-6/keystone
```
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    }),
+  },
+};
+```
+
+The key of the `lists` object are used as the name of the list. The key of the `fields` object are used as the name for the fields.
+
+## Starting your new app
+
+`yarn keystone dev`
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:84]:
````suggestion
```
yarn keystone dev
```
````

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+When you run keystone, it will use this file's export to load the config. If we try to run `yarn keystone dev` it will not work.
+
+> If you want to use typescript, you can use keystone.ts instead.
+
+### Setting up a Database
+
+For our first project, we're going to use `sqlite` as we can do this without any further database setup. The config for this is below:
+
+```js
+export default {
```
Comment (mitchellhamilton) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:44]: Could we encourage using the `config` function?

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+yarn add @keystone-6/keystone
+```
+
+## Setting up Keystone
+
+There are two bits of keystone's configuration that are required to start our project: the `lists`, and the `db`. The `lists` define
+she shape of the information we want to save in our database, while the `db` defines how we will connect to our database. For now though
+let us set up our `keystone.js` file
+
+`touch keystone.js`
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:31]: Probably don't need to `touch`,   I can't think of many editors that won't create the file if it doesn't exist when saving

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+> In this lesson we will install the keystone package, create our first list, and get Keystone running
+
+## Installing the packages you need
+
+We will be using `yarn` for this, however you can use `npm` if you would prefer.
+
+The first thing we are going to do is set ourselves up a workspace for our new project. Let's make ourselves a new folder and initialise yarn.
+
+```sh
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:13]:
```suggestion
```bash
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+mkdir keystone-learning
+cd keystone-learning
+yarn init
+```
+
+After this, let's add the keystone package
+
+```sh
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:21]:
```suggestion
```bash
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    }),
+  },
+};
+```
+
+The key of the `lists` object are used as the name of the list. The key of the `fields` object are used as the name for the fields.
+
+## Starting your new app
+
+```sh
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:86]:
```suggestion
```bash
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx (hunk 3 above)
Comment (Noviny) [reply docs/pages/docs/walkthroughs/lesson-1.mdx:46]: Sure

File: docs/components/docs/WalkthroughsList.tsx [hunk 8]
```diff
@@ -29,6 +29,10 @@ export function Walkthroughs() {
         Learn how to run Keystone in the same folder as your frontend code and commit everything to
         Git. You end up with a queryable GraphQL endpoint running live on Vercel for free.
```
Comment (dcousens) [inline docs/components/docs/WalkthroughsList.tsx:30]:
```suggestion
        Git. You end up with a queryable GraphQL endpoint running live on Vercel.
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 9]
```diff
@@ -0,0 +1,100 @@
+import { Markdown } from '../../../components/Markdown';
+
+# Lesson 1: Setting up Keystone
+
+> In this lesson we will install the keystone package, create our first list, and get Keystone running
+
+## Installing the packages you need
+
+We will be using `yarn` for this, however you can use `npm` if you would prefer.
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:9]:
```suggestion
For our packages,  we will be using `yarn`,  but you can use `npm` or any other package manager that you prefer.
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+The first thing we are going to do is set ourselves up a workspace for our new project. Let's make ourselves a new folder and initialise yarn.
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:11]:
```suggestion
The first thing we are going to do is set ourselves up a workspace for our new project. Let's make ourselves a new folder and initialise `yarn`.
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+```bash
+mkdir keystone-learning
+cd keystone-learning
+yarn init
+```
+
+After this, let's add the keystone package
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:19]:
```suggestion
After this, let's add the Keystone package
```

File: docs/pages/docs/walkthroughs/lesson-1.mdx [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+```bash
+yarn add @keystone-6/keystone
```
Comment (dcousens) [inline docs/pages/docs/walkthroughs/lesson-1.mdx:22]:
```suggestion
yarn add @keystone-6/core
```

**Review by raveling — APPROVED** [review]
LGTM!

**Pull request merged**